geomesa-gs-plugin/geomesa-hbase-gs-plugin/pom.xml

```diff
@@ -13,7 +13,7 @@
     <parent>
         <artifactId>geomesa-gs-plugin</artifactId>
         <groupId>org.locationtech.geomesa</groupId>
-        <version>1.2.0-SNAPSHOT</version>
+        <version>1.2.1-SNAPSHOT</version>
```

Comment:
hmm should I also add back in bigtable? then I can get rid of the profiles in this pom